### ISSUE TITLE
Add beginner-section and advanced-section

### ISSUE THREAD

**Issue by charandeepsinghb** [charandeepsinghb/learn-code-contribute#10]
Create new pages in the components folder:
beginner-section.html
advanced-section.html

beginner-section.html -> contains:
All the links to the navigation items with anchor (\<a\>) tags
like -->
```
<a href="https://charandeepsinghb.github.io/learn-code-contribute/#/components/basicsofgit"></a>
....
<a href="https://charandeepsinghb.github.io/learn-code-contribute/#/components/howcontribute"></a>
```

**Comment by AjinkyaBorate** [comment]
@chararndeepsinghb I would like to contribute to this repo

**Comment by charandeepsinghb** [comment]
@Ajinkya2105 you can add a PR for https://github.com/charandeepsinghb/learn-code-contribute/issues/11
The best code will be merged.
Appreciations 👍

**Comment by happyday123123** [comment]
@charandeepsinghb I have completed this task. Could my PR be accepted ?

**Comment by AjinkyaBorate** [comment]
@charandeepsinghb  i got your issue and i would like to work on  #10

**Comment by charandeepsinghb** [comment]
Already closed by #13

**Comment by AjinkyaBorate** [comment]
ok

**Comment by AjinkyaBorate** [comment]
> Already closed by #13

ok

**Comment by charandeepsinghb** [comment]
@AjinkyaBorate you can work on the #17
